numpy/core/_ufunc_config.py

```diff
@@ -344,7 +345,11 @@ class errstate(contextlib.ContextDecorator):
     ..  versionchanged:: 1.17.0
         `errstate` is also usable as a function decorator, saving
         a level of indentation if an entire function is wrapped.
-        See :py:class:`contextlib.ContextDecorator` for more information.
+
+    .. versionchanged:: 2.0
```

Comment:
Just a nit, but assuming this will be backported for patch release I think this should be 1.25 (unless I'm misunderstanding!)

Reply:
no, this is only in the nightlies/main.